catalog/MDCCatalog/MDCCatalogTiles.m

```diff
@@ -4098,12 +4109,19 @@ void MDCCatalogDrawTabsTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme)
   CGColorSpaceRelease(colorSpace);
 }
 
-static NSString* const TextFieldFontName = @"RobotoMono-Medium";
-
 void MDCCatalogDrawTextFieldTile(CGRect frame, NSObject<MDCColorScheme> *colorScheme) {
   UIColor* fillColor = colorScheme.primaryLightColor;
   UIColor* textForeground = colorScheme.primaryColor;
   
+   UIFont *(^PalettesFont)(void) = ^(void){
```

Comment:
Same feedback here - it's probably best to stick with a basic `UIFont *` and if/else.